### PR TITLE
chore(templates): add github template for issues and pull request

### PR DESCRIPTION
**Pull request by EmanuelCampos**
copied from llama-index with small adjustments

note: there's not documentation here, but I kept to refer to readmes, notebooks and etc...

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of Change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
```
Comment (jerryjliu) [inline .github/pull_request_template.md:12]: nit: i'd rename this to "New Loader/Tool" and move to the top

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of Change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
```
Comment (jerryjliu) [inline .github/pull_request_template.md:11]: nit: would rename this to "Bug fix / Smaller change"

**Review by jerryjliu — APPROVED** [review]
thanks!! some very minor comments

**Comment by EmanuelCampos** [comment]
@jerryjliu changes made, merging it

**Pull request merged**